roles/lib_utils/src/class/yedit.py

```diff
@@ -194,16 +193,16 @@ def write(self):
         if self.backup and self.file_exists():
             shutil.copy(self.filename, self.filename + '.orig')
 
-        if hasattr(yaml, 'RoundTripDumper'):
-            # AUDIT:no-member makes sense here due to ruamel.YAML/PyYAML usage
-            # pylint: disable=no-member
-            if hasattr(self.yaml_dict, 'fa'):
-                self.yaml_dict.fa.set_block_style()
+        # Try to set format attributes if supported
+        try:
```

Comment:
Oddly enough, at least with python3 using exceptions for this isn't any less performant than hasattr, because hasattr just swallows the AttributeError.